persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/StructuralVariantMapper.xml

```diff
@@ -0,0 +1,90 @@
+        structural_variant.NORMAL_SPLIT_READ_COUNT AS "${prefix}normalSplitReadCount",
+        structural_variant.TUMOR_SPLIT_READ_COUNT AS "${prefix}tumorSplitReadCount",
+        structural_variant.ANNOTATION AS "${prefix}annotation",
+        structural_variant.BREAKPOINT_TYPE AS "${prefix}breakpointType",
+        structural_variant.CENTER AS "${prefix}center",
+        structural_variant.CONNECTION_TYPE AS "${prefix}connectionType",
+        structural_variant.EVENT_INFO AS "${prefix}eventInfo",
+        structural_variant.CLASS AS "${prefix}variantClass",
+        structural_variant.LENGTH AS "${prefix}length",
+        structural_variant.COMMENTS AS "${prefix}comments",
+        structural_variant.EXTERNAL_ANNOTATION AS "${prefix}externalAnnotation",
+        structural_variant.DRIVER_FILTER AS "${prefix}driverFilter",
+        structural_variant.DRIVER_FILTER_ANNOTATION AS "${prefix}driverFilterAnn",
+        structural_variant.DRIVER_TIERS_FILTER AS "${prefix}driverTiersFilter",
+        structural_variant.DRIVER_TIERS_FILTER_ANNOTATION AS "${prefix}driverTiersFilterAnn"
+    </sql>
+
+    <select id="fetchStructuralVariants" resultType="org.cbioportal.model.StructuralVariant">
+        SELECT 
+        <include refid="select">
+            <property name="prefix" value=""/>
+        </include> 
+        FROM structural_variant
+        JOIN genetic_profile ON structural_variant.GENETIC_PROFILE_ID = genetic_profile.GENETIC_PROFILE_ID
+        JOIN gene gene1 ON structural_variant.SITE1_ENTREZ_GENE_ID = gene1.ENTREZ_GENE_ID
+        JOIN gene gene2 ON structural_variant.SITE2_ENTREZ_GENE_ID = gene2.ENTREZ_GENE_ID
+        <if test="studyIds.size() > 0 and sampleIds.size() > 0">
+            JOIN sample ON structural_variant.SAMPLE_ID = sample.INTERNAL_ID
+            JOIN patient ON sample.patient_ID = patient.INTERNAL_ID
+            JOIN cancer_study ON patient.CANCER_STUDY_ID = cancer_study.CANCER_STUDY_ID
+        </if>
+        <where>
+            genetic_profile.STABLE_ID IN 
+            <foreach item="item" collection="geneticProfileStableIds" open="(" separator="," close=")">
+                #{item}
+            </foreach>
+            AND (gene1.HUGO_GENE_SYMBOL in 
+            <foreach item="item" collection="hugoGeneSymbols" open="(" separator="," close=")">
+                #{item}
+            </foreach>
+            OR gene2.HUGO_GENE_SYMBOL IN
+            <foreach item="item" collection="hugoGeneSymbols" open="(" separator="," close=")">
+                #{item}
+            </foreach>
+            )
+            <if test="studyIds.size() > 0 and sampleIds.size() > 0">
+            AND (cancer_study.CANCER_STUDY_IDENTIFIER, sample.STABLE_ID) IN
+            <foreach index="i" collection="sampleIds" open="(" separator="," close=")"> 
+                (#{studyIds[${i}]}, #{sampleIds[${i}]}) 
+            </foreach>
+        </if>
+        </where>
```

Comment:
please also add an `ORDER BY` clause to your query. This can be `ORDER BY gene1.HUGO_GENE_SYMBOL, gene2.HUGO_GENE_SYMBOL`